### ISSUE TITLE
As a developer of a tool that generates RO-Crates, I want to validate the created RO-Crates as part of my test suite

### ISSUE THREAD

**Issue by elichad** [eScienceLab/RO-Crate_Issues#52]
Useful for testing if RO-Crates generated by third-party tools are valid - e.g. a Workflow Run Crate generated by Galaxy or snakemake.

Likely to take the form of a command-line tool

- [x] Evaluate [rocrate-validator](https://github.com/crs4/rocrate-validator)
- [x] Implement it in the test suite for galaxy or cwlrunner
- [x] Document somehow (e.g. by example) the approach to take

**Comment by stain** [comment]
With pytest or github action examples

**Comment by OliverWoolland** [comment]
https://github.com/crs4/rocrate-validator might help?

**Comment by ianhinder** [comment]
Next task here is for Oliver to evaluate [rocrate-validator](https://github.com/crs4/rocrate-validator).

**Comment by ianhinder** [comment]
Oliver is starting to develop in Galaxy, and will plug in the validator.

**Comment by stain** [comment]
rocrate-validator programmatic use needs to be improved to be used for testing and in Cratey-validator.  See crs4/rocrate-validator#40

**Comment by ianhinder** [comment]
A [PR](https://github.com/crs4/rocrate-validator/pull/42/files) has been raised. We will take a look at this.

**Comment by ianhinder** [comment]
This is on hold, because we are waiting for RunCrate to work with Galaxy, so that we can get usable provenance from Galaxy. Current RO-Crate output from Galaxy does not pass validation, so we want to replace it.

**Comment by stain** [comment]
@OliverWoolland ran validation offline not added to Galaxy tool base. Re-visit after #91

**Comment by elichad** [comment]
rocrate-validator now has this capability and documentation: https://rocrate-validator.readthedocs.io/en/latest/2_usage_cli/
And I have a draft PR for adding it to the Galaxy test suite: https://github.com/galaxyproject/galaxy/pull/19846 . It was straightforward to implement using the documentation above.

I'm going to close this as the user story is fulfilled.
